apps/backend/src/libs/plugins/authorization/authorization.plugin.ts

```diff
@@ -28,15 +28,22 @@ const authorization = fp<Options>((fastify, options, done) => {
 			}
 		}
 
-		const jwtToken = request.headers["authorization"];
+		const authHeader = request.headers["authorization"];
 
-		if (!jwtToken) {
+		if (!authHeader) {
```

Comment:
let's move this check here
```suggestion
		if (!authHeader || !authHeader.startsWith(BEARER_PREFIX)) {
```